### PR TITLE
DataGrid: Update Virtual Scrolling description

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Review by Alyar666 — DISMISSED** [review]

**Review by AlexJameson — DISMISSED** [review]

**Review by pavel-2020 — DISMISSED** [review]

File: JSDemos/Demos/DataGrid/VirtualScrolling/description.md [hunk 1]
```diff
@@ -1 +1,5 @@
-The **DataGrid** widget supports virtual record scrolling, allowing end users to&nbsp;navigate the contents of&nbsp;the widget by&nbsp;using its vertical scrollbar. In&nbsp;this demo, the **DataGrid** is&nbsp;connected to&nbsp;100,000 records and virtual scroll mode has been enabled to&nbsp;simplify record navigation and eliminate the use of&nbsp;data pagers.
\ No newline at end of file
+Virtual scrolling enables users to scroll grid records in order or jump to any record. In this mode, records are loaded when they get into the viewport and removed once they leave it.
```
Comment (Abadzhev) [inline JSDemos/Demos/DataGrid/VirtualScrolling/description.md]:
```suggestion
If the DataGrid component works with a large dataset, you can enable virtual scrolling to optimize data loading and navigation. The component calculates the overall number of visible rows and displays a scrollbar so users can jump to any location. When users release the scroll thumb, the control loads the records to be displayed in the viewport and removes other rows from memory. 
```

File: JSDemos/Demos/DataGrid/VirtualScrolling/description.md [hunk 2]
```diff
@@ -1 +1,5 @@
-The **DataGrid** widget supports virtual record scrolling, allowing end users to&nbsp;navigate the contents of&nbsp;the widget by&nbsp;using its vertical scrollbar. In&nbsp;this demo, the **DataGrid** is&nbsp;connected to&nbsp;100,000 records and virtual scroll mode has been enabled to&nbsp;simplify record navigation and eliminate the use of&nbsp;data pagers.
\ No newline at end of file
+If the **DataGrid** component works with a large dataset, you can enable virtual scrolling to optimize data loading and navigation. The component calculates the overall number of visible rows and displays a scrollbar so users can jump to any location. When users release the scroll thumb, the control loads the records to be displayed in the viewport and removes other rows from memory.
```
Comment (albertov05) [inline JSDemos/Demos/DataGrid/VirtualScrolling/description.md]: I'd like to make this more specific than "works with" and "jump to any location". Could we, for example, say:

```suggestion
If the **DataGrid** component is bound to a large dataset, you can enable the virtual scroll feature to optimize data load times and improve user navigation. The component calculates the overall number of visible rows and displays a scrollbar that allows users to navigate to any section of rows. When users release the scroll thumb, the control loads records to be displayed in the viewport and removes other rows from memory. 
```

File: JSDemos/Demos/DataGrid/VirtualScrolling/description.md [hunk 3]
```diff
@@ -1 +1,5 @@
-The **DataGrid** widget supports virtual record scrolling, allowing end users to&nbsp;navigate the contents of&nbsp;the widget by&nbsp;using its vertical scrollbar. In&nbsp;this demo, the **DataGrid** is&nbsp;connected to&nbsp;100,000 records and virtual scroll mode has been enabled to&nbsp;simplify record navigation and eliminate the use of&nbsp;data pagers.
\ No newline at end of file
+If the **DataGrid** component works with a large dataset, you can enable virtual scrolling to optimize data loading and navigation. The component calculates the overall number of visible rows and displays a scrollbar so users can jump to any location. When users release the scroll thumb, the control loads the records to be displayed in the viewport and removes other rows from memory. 
+
+To enable virtual scrolling, set the **scrolling**.[mode](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/scrolling/#mode) to *"virtual"*.
```
Comment (albertov05) [inline JSDemos/Demos/DataGrid/VirtualScrolling/description.md]:
```suggestion
To allow users to scroll the **DataGrid** virtually, set the **scrolling**.[mode](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/scrolling/#mode) to *"virtual"*.
```

File: JSDemos/Demos/DataGrid/VirtualScrolling/description.md [hunk 4]
```diff
@@ -1 +1,5 @@
...
+
+In this demo, the **DataGrid** is bound to a local dataset of 100,000 records. Thanks to virtual scrolling, you can scroll to any record without lag.
```
Comment (albertov05) [inline JSDemos/Demos/DataGrid/VirtualScrolling/description.md]: I'd like to revise the second sentence. What do you think of the following?

```suggestion
In this demo, the **DataGrid** is bound to a local dataset of 100,000 records. You can drag the scrollbar on the right to see that records within the viewport are updated immediately.
```

**Pull request merged**